web-client/src/ustc-ui/Icon/ConsolidatedCaseIcon.tsx

```diff
@@ -7,19 +7,24 @@ import React from 'react';
  * @param {object} caseItem the case to be passed to the FontAwesomeIcon
  * @returns {object} a react component
  */
-export const ConsolidatedCaseIcon = ({ caseItem }) => {
+export const ConsolidatedCaseIcon = ({
+  consolidatedIconTooltipText,
+  inConsolidatedGroup,
+  showLeadCaseIcon,
+}: {
+  inConsolidatedGroup: string;
```

Comment:
should `inConsolidatedGroup` be a string if it's used as a truthy value on line 20? This would get outputted. Perhaps a `boolean` is better?